index.Rmd

```diff
@@ -51,6 +51,17 @@ These have not yet been run as live sessions, but you can work through the mater
 
 * [Introduction to Python](https://github.com/moj-analytical-services/intro-to-python)
 * [AWS Tooling demos](https://github.com/moj-analytical-services/mojap-aws-tools-demo)
+* [Pyspark in AP](https://github.com/moj-analytical-services/pyspark-in-AP-guidance)
```

Comment:
This is definitely useful to link to but it isn't really training, maybe put it under books?

Reply:
Maybe it is training actually, perhaps ignore me!

Reply:
Thanks both - the current structure of this document is such that I think the books should be in the additional resources section and the coffee and coding session in a separate coffee and coding chapter. We will I think shortly be putting together skills lists and then linking to training so we can then have a chapter focusing on the Python pathway that will link up everything for this.

Reply:
As it's been a while I've added the content to the internal training and main additional resources section  as appropriate.